l10n_it_fiscal_payment_term/migrations/10.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2017-2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def migrate(cr, version):
```

Comment:
@sergiocorato aggiungendo il ```pre_init_hook``` questo script non diventa superfluo?

Reply:
@eLBati no, facendo una prova mi vengono duplicati (gli script pre o post init_hook non mi sembra vengano richiamati in fase di aggiornamento, ti risulta?)

Reply:
ecco come risulta sul database togliendo lo script pre-migration:

testit=> select * from ir_model_data where model = 'fatturapa.payment_term';
  id   | create_uid |        create_date         |         write_date         | write_uid | noupdate |      name      |      date_init      |     date_update     |           module            |         model          | res_id 
-------+------------+----------------------------+----------------------------+-----------+----------+----------------+---------------------+---------------------+-----------------------------+------------------------+--------
 23704 |          1 | 2018-09-03 07:47:10.752767 | 2018-09-03 07:47:10.752767 |         1 | t        | fatturapa_TP01 | 2018-09-03 07:47:10 | 2018-09-03 07:47:10 | l10n_it_fatturapa           | fatturapa.payment_term |      1
 23705 |          1 | 2018-09-03 07:47:10.752767 | 2018-09-03 07:47:10.752767 |         1 | t        | fatturapa_tp02 | 2018-09-03 07:47:10 | 2018-09-03 07:47:10 | l10n_it_fatturapa           | fatturapa.payment_term |      2
 23706 |          1 | 2018-09-03 07:47:10.752767 | 2018-09-03 07:47:10.752767 |         1 | t        | fatturapa_tp03 | 2018-09-03 07:47:10 | 2018-09-03 07:47:10 | l10n_it_fatturapa           | fatturapa.payment_term |      3
 23707 |          1 | 2018-09-03 07:47:10.752767 | 2018-09-03 07:47:10.752767 |         1 | t        | fatturapa_tp04 | 2018-09-03 07:47:10 | 2018-09-03 07:47:10 | l10n_it_fatturapa           | fatturapa.payment_term |      4
 27021 |          1 | 2018-09-03 08:34:00.684457 | 2018-09-03 08:34:00.684457 |         1 | t        | fatturapa_TP01 | 2018-09-03 08:34:00 | 2018-09-03 08:34:00 | l10n_it_fiscal_payment_term | fatturapa.payment_term |      5
 27022 |          1 | 2018-09-03 08:34:00.684457 | 2018-09-03 08:34:00.684457 |         1 | t        | fatturapa_tp02 | 2018-09-03 08:34:00 | 2018-09-03 08:34:00 | l10n_it_fiscal_payment_term | fatturapa.payment_term |      6
 27023 |          1 | 2018-09-03 08:34:00.684457 | 2018-09-03 08:34:00.684457 |         1 | t        | fatturapa_tp03 | 2018-09-03 08:34:00 | 2018-09-03 08:34:00 | l10n_it_fiscal_payment_term | fatturapa.payment_term |      7
 27024 |          1 | 2018-09-03 08:34:00.684457 | 2018-09-03 08:34:00.684457 |         1 | t        | fatturapa_tp04 | 2018-09-03 08:34:00 | 2018-09-03 08:34:00 | l10n_it_fiscal_payment_term | fatturapa.payment_term |      8